kernel/shared/src/main/scala/cats/effect/kernel/Resource.scala

```diff
@@ -1305,8 +1305,8 @@ private[effect] trait ResourceHOInstances4 extends ResourceHOInstances5 {
   @deprecated(
     "Bring an implicit MonadCancelThrow[F] into scope to get the fixed Resource instance",
```

Comment:
Maybe we can slightly revise this.

```suggestion
    "Use catsEffectMonadCancelForResource",
```